### ISSUE TITLE
case issue in opening .xlsx

### ISSUE THREAD

**Issue by Tux** [doy/spreadsheet-parsexlsx#57]
Someone complained on Spreadsheet::Read being unable to open an xlsx, which I deduced to a problem in entry-name casing in the xlsx. The provided file looks like

```
Archive:  ../Spreadsheet-Read/sandbox/price.xlsx
  Length      Date    Time    Name
---------  ---------- -----   ----
     1016  2016-04-29 21:42   [Content_Types].xml
   473496  2016-04-29 21:42   xl/SharedStrings.xml
     6824  2016-04-29 21:42   xl/styles.xml
      592  2016-04-29 21:42   xl/workbook.xml
      207  2016-04-29 21:42   xl/drawings/drawing1.xml
      145  2016-04-29 21:42   xl/drawings/_rels/drawing1.xml.rels
  2118890  2016-04-29 21:42   xl/worksheets/sheet1.xml
      307  2016-04-29 21:42   xl/worksheets/_rels/sheet1.xml.rels
      580  2016-04-29 21:42   xl/_rels/workbook.xml.rels
      305  2016-04-29 21:42   _rels/.rels
---------                     -------
  2602362                     10 files
```
causing this error:
```
Can't call method "contents" on an undefined value at /pro/3gl/CPAN/Spreadsheet-ParseXLSX-git/blib/lib/Spreadsheet/ParseXLSX.pm line 795.
```
because ParseXLSX expects xl/sharedStrings.xml (shared with a lower-case s)

**Comment by doy** [comment]
Do you have an example of a file like this? Or could I at least see the contents of xl/_rels/workbook.xml.rels?

**Comment by Tux** [comment]
[price.xlsx](https://github.com/doy/spreadsheet-parsexlsx/files/276285/price.xlsx)

That is what I have been sent, and what I used to find out what was happening

**Comment by doy** [comment]
Fixed in c7c05c5.
